src/mongocxx/test/index_view.cpp

```diff
@@ -174,6 +174,9 @@ TEST_CASE("create_one", "[index_view]") {
     }
 
     SECTION("commitQuorum option") {
+        if (test_util::get_topology(mongodb_client) == "single")
```

Comment:
Can you add a warning for consistency with other skips? `WARN("skip: commitQuorum option requires a replica set");`

Reply:
Done.